docs/send-data/opentelemetry-collector/install-collector-macos.md

```diff
@@ -28,10 +28,21 @@ Versions Supported
 
 You can install our OpenTelemetry Collector using one of the following methods:
 
+* [UI Installation](#ui-installation)
 * [Install script](#install-script)
-* [UI Installation via App Catalog](#ui-installation-via-app-catalog)
 * [Manual step-by-step installation](#manual-step-by-step-installation)
 
+### UI Installation
+
+1. In Sumo Logic, select **Manage Data** > **Collection** > **OpenTelemetry Collection**.
+1. On the OpenTelemetry Collection page, click **Add Collector**.
+1. On the left panel select **macOs** as the platform.<br/> <img src={useBaseUrl('img/send-data/opentelemetry-collector/macOs.png')} alt="macOs-terminal" style={{border: '1px solid black'}} width="900"/>
```

Comment:
```suggestion
1. On the left panel select **macOS** as the platform.<br/> <img src={useBaseUrl('img/send-data/opentelemetry-collector/macOs.png')} alt="macOs-terminal" style={{border: '1px solid black'}} width="900"/>
```

Reply:
It is an UI term, so maintaining macOs as it is.